pkgs/swarmauri/swarmauri/llms/concrete/PlayHTModel.py

```diff
@@ -0,0 +1,198 @@
+import asyncio
+import io
+import os
+
+import aiohttp
+import requests
+from typing import List, Literal, Dict
+
+from pydantic import Field
+from swarmauri.llms.base.LLMBase import LLMBase
+
+
+class PlayHTModel(LLMBase):
+    """
+    Play.ht TTS class for text-to-speech synthesis
+    """
+
+    api_key: str
+    user_id: str
+    allowed_models: List[str] = Field(default=["Play3.0-mini", "PlayHT2.0-turbo"])
+    allowed_voices: List[str] = Field(
+        default=[
+            "s3://voice-cloning-zero-shot/775ae416-49bb-4fb6-bd45-740f205d20a1/jennifersaad/manifest.json"
+        ]
+    )
+    voice: str = (
+        "s3://voice-cloning-zero-shot/775ae416-49bb-4fb6-bd45-740f205d20a1/jennifersaad/manifest.json"
+    )
```

Comment:
we must read the json file and get the array of voices supported

Reply:
After a thorough check i discovered that voices are specific to a particular voice engine model, which mean when a voice that  isn't linked to particular voice model is passed, it throws a `500` error. [**solution**] I  linked a list of all voices to its specific voice engine model. then created a function that validates it when instantiating the class.

Reply:
Again, the `api` that returns a list of allowed voices, returns voices specific to just two voice engine model `PlayHT2.0` and `PlayHT1.0`.  However PlayHT supports 4 models `["Play3.0-mini", "PlayHT2.0-turbo", "PlayHT1.0", "PlayHT2.0"]`. Now i did a further test, i was able to discover that all voices of `PlayHT2.0` works across all models, but only voices of  `PlayHT1.0` works for its model  `PlayHT1.0`